creator-node/src/blacklistManager.js

```diff
@@ -214,20 +257,58 @@ class BlacklistManager {
   /**
    * Retrieves all the deduped CIDs from the params and builds a mapping to <trackId: segments> for explicit trackIds (i.e. trackIds from table, not tracks belonging to users).
    * @param {Object} param
-   * @param {number[]} param.allTrackIds all the trackIds to find CIDs for
-   * @param {Set<number>} param.explicitTrackIds explicit trackIds from table. Used to create mapping of <trackId: segments> in redis
+   * @param {number[]} param.allTrackIds all the trackIds to find CIDs for (explictly blacklisted tracks and tracks from blacklisted users)
```

Comment:
this is just adding back in the fn that was removed here https://github.com/AudiusProject/audius-protocol/pull/3250

Reply:
and also renaming vars to be more clear